src/icons/review.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
Weren't we talking about trying to clean up/optimize (ie, svgo) our SVGs?

Reply:
Are you talking about doing that as part of the build? This is just one I received from the designer (that is, not exported from Figma - so, better?)

Reply:
I guess I was expecting the designers to provide optimized SVGs?